app/Integrations/GitHub/GitHubPlugin.php

```
@@ -0,0 +1,423 @@
+        $config = $integration->configuration ?? [];
+        $repositories = $config['repositories'] ?? [];
+        $events = $config['events'] ?? ['push', 'pull_request'];
+        
+        foreach ($repositories as $repo) {
+            $this->fetchRepositoryEvents($repo, $events, $integration);
+        }
+    }
+    
+    protected function fetchRepositoryEvents(string $repo, array $events, Integration $integration): void
+    {
+        foreach ($events as $eventType) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$eventType'. (Unused Code Rules)

(UnusedLocalVariable)